README.md

```diff
@@ -19,7 +19,13 @@ For the v2.x *Flutter* docs app, visit [here](https://github.com/pngme/sample-an
    - periodically send SMS data to pngme's data processing pipeline
 2. The SDK supports Android API level 16+
 3. The SDK exposes three methods: a main entrypoint method, and two helper methods
-4. Using the SDK requires an SDK `clientKey`. Sign up and get started _for free_ at the [Pngme admin webconsole](https://admin.pngme.com)
+4. Using the SDK requires an **SDK Token** 
+   * To retrieve your **SDK Token**, sign up and **get started _for free_** at the [Pngme admin webconsole](https://admin.pngme.com)
+   * Once you have created your organization, navigate to `Keys` in the webconsole and copy the SDK Token for the environment you want to use
+
+ ![webconsole keys screen](.docs/webconsole_keys.png)
+
+   > Notice how **pngme** provides a `production` environment meant to receive real data from real final users an a `test` environment for testing and evaluation purposes
```

Comment:
typo ;)

Reply:
This one took me a couple of minutes XD Thanks!